src/management/api/portal/general/apiPortal.controller.ts

```diff
@@ -19,6 +19,7 @@ import UserService from '../../../../services/user.service';
 import {QualityMetrics} from '../../../../entities/qualityMetrics';
 import ApiService from '../../../../services/api.service';
 import PolicyService from '../../../../services/policy.service';
+import PortalConfigService from "../../../../services/portalConfig.service";
```

Comment:
" should be '

Reply:
npm run lint
npm run lint:fix

and more globally mvn clean install (which runs linter)

Reply:
resolved